### ISSUE TITLE
Data Augmentation using Pre-trained Transformer Models

### ISSUE THREAD

**Issue by aisolab** [aisolab/paper-log#9]
## 어떤 내용의 논문인가요? 👋
- pre-trained transformer model (e.g. BERT, GPT2, BART 등)을 이용 text data augmentation을 하는 방법론을 비교 대조한 논문

## Abstract (요약) 🕵🏻‍♂️

> Language model based pre-trained models such as BERT have provided significant gains across different NLP tasks. In this paper, we study different types of pre-trained transformer based models such as auto-regressive models (GPT-2), auto-encoder models (BERT), and seq2seq models (BART) for conditional data augmentation. We show that prepending the class labels to text sequences provides a simple yet effective way to condition the pre-trained models for data augmentation. On three classification benchmarks, pre-trained Seq2Seq model outperforms other models. Further, we explore how different pretrained model based data augmentation differs in terms of data diversity, and how well such methods preserve the class-label information.

## 이 논문을 읽어서 무엇을 배울 수 있는지 알려주세요! 🤔

- pre-trained transformer model을 이용하여 text data augmentation을 하는 방법
- semantic fidelity, text diversity를 measure하는 방법

## 같이 읽어보면 좋을 만한 글이나 이슈가 있을까요?
- [Contextual Augmentation: Data Augmentation by Words with Paradigmatic Relations](https://arxiv.org/abs/1805.06201)
- [Conditional BERT Contextual Augmentation](https://arxiv.org/abs/1812.06705)
## 레퍼런스의 URL을 알려주세요! 🔗
https://arxiv.org/abs/2003.02245

**Comment by aisolab** [comment]
https://aisolab.github.io/paper/Data-Augmentation-using-Pre-trained-Transformer-Models/
